imports/ui/TaskConfigInputs.tsx

```diff
@@ -95,6 +104,7 @@ export function TaskConfigInputs({
             setPlannedDate(newPlannedDate);
           }}
         />
```

Comment:
The button looks like an oval when it's included here. Also it pushes past the margin. Can you figure out what to do with the styles to fix that?

Reply:
I still don't really understand why it was becoming an oval, but wrapping it in a div is the easiest fix I found.